.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn lint-staged
+npm run-script lint-staged
```

Comment:
```suggestion
yarn run lint-staged
```
If you want to do that more specific.
We need to use `yarn` to be more consistent.